packages/teams-js/src/public/meeting.ts

```diff
@@ -597,6 +620,39 @@ export namespace meeting {
     );
   }
 
+  /**
+   * @hidden
+   * Allows an app to get the verbose meeting details for the meeting
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export async function getMeetingDetailsVerbose(): Promise<IMeetingDetailsResponse> {
+    ensureInitialized(
+      runtime,
+      FrameContexts.sidePanel,
+      FrameContexts.meetingStage,
+      FrameContexts.settings,
+      FrameContexts.content,
+    );
+
+    const shouldGetVerboseDetails = true;
+    const response = (await sendAndHandleSdkError(
+      getApiVersionTag(ApiVersionNumber.V_2, ApiName.Meeting_GetMeetingDetailsVerbose),
```

Comment:
@TrevorJoelHarris Do you by any chance know which ApiVersionNumber we should be using here?

Reply:
this one is correct, because you are adding this function for the first time to version 2.* of the TeamsJS library.